### DISCUSSION TITLE
Enhancing Dataset Management: Introducing Folders and Tags for Better Organization and Filtering

### DISCUSSION THREAD

**Discussion by Schnurres** [apache/superset#24351] [Ideas]
Datasets can become very large, often consisting of dozens or even 100+ columns and a high number of metrics. Although there is a search function, it still gets very cluttered. It would be great if there were a way to organize them, for example, by placing them in folders or assigning tags that can be easily filtered.

This would enhance the clarity and improve the workflow when working with large datasets.

**Comment by lf-floriandin** [comment]
I like that. Maybe even folder/tags for Datasets, Charts, Dashboards etc.
